charts/brigade-metrics/Chart.yaml

```diff
@@ -5,6 +5,7 @@ description: |
 type: application
 version: 0.1.0
 appVersion: v0.1.0
+icon: https://raw.githubusercontent.com/brigadecore/metrics/main/logo.png
```

Comment:
Does `brigadecore/metrics` redirect?  Should be `brigadecore/brigade-metrics`, right?

Reply:
It does redirect because "brigade-metrics" was formerly just "metrics," but this _is_ an oversight, as the repo name should be brigade-metrics.